prow/config.yaml

```diff
@@ -12300,7 +12816,9 @@ periodics:
       secret:
         defaultMode: 256
         secretName: ssh-key-secret
+
 - interval: 6h
+  agent: kubernetes
```

Comment:
why are all of these ordered before the name? shouldn't they retain the template order?